packages/central-server/src/apiV2/userAccounts/RegisterUserAccounts.js

```diff
@@ -54,7 +54,9 @@ export class RegisterUserAccounts extends CreateUserAccounts {
       email: { comparisonValue: emailAddress, comparator: 'ilike' },
     });
     if (existingUsers.length > 0) {
-      throw new UnauthenticatedError('Existing user found with same email address.');
+      throw new UnauthenticatedError(
+        "An account already exists with this email. Please log in or click 'forgot password' if you have forgotten your password",
```

Comment:
Much nicer message!

Nitpick, but would be nice to [educate those quotes](https://practicaltypography.com/straight-and-curly-quotes.html):

```suggestion
        'An account already exists with this email. Please log in or click ’Forgot password?‘ if you have forgotten your password',
```